frontend/add-client/demographic-information.component.tsx

```diff
@@ -109,7 +115,22 @@ export default function DemographicInformation(props: StepComponentProps) {
         </div>
         <div>
           <label>
-            <span>Are they a eligible to vote?</span>
+            <span>Number household dependents under age 18</span>
+            <input
+              type="number"
+              value={householdSizeChildren}
+              onChange={evt =>
+                setHouseholdSizeChildren(Number(evt.target.value))
+              }
+              required
+              min="0"
```

Comment:
nitpick (not important): you can actually do `min={0}` and `max={30}`, passing in numbers instead of strings.

This is because dom elements have two types of data: html attributes and javascript properties. The attributes all have to be strings, but the properties can by other data types. And it turns out that react almost always (only aria- and custom elements are exceptions) sets properties, not attributes. And the properties can be numbers instead of strings.